tools/orca/src/kani.rs

```diff
@@ -99,9 +99,16 @@ impl KaniHttpServer {
         all_entities: &HashMap<Uuid, Entity>,
     ) -> Result<(), ()> {
         // Create all the accounts and groups
-        for u in targets {
-            let e = all_entities.get(u).unwrap();
-            match e {
+        let num_uuids = targets.len();
+        let mut current_slice = 1;
+        info!("Have to do {} uuids", num_uuids);
+        for (index, uuid) in targets.iter().enumerate() {
```

Comment:
I wonder if it might be easier to do:

```
for (pct, chunk) in targets.chunks(targets.len() / 10).enumerate() {
    for uuid in chunk {
    }
    info!("{}% complete" pct * 10);
}
```

Reply:
That'd mean having to pull the targets out into a new vec as well, because targets is a HashBrown::hash_set, and it just gets messier from there